test/unit/client/simple-query-tests.js

```diff
@@ -120,4 +120,25 @@ test('executing query', function () {
       })
     })
   })
+
+  test('handles errors', function () {
+    var client = helper.client()
+    var error = new Error('Client was passed a null or undefined query')
```

Comment:
Is this variable unused?

Reply:
Yep, must have missed it when I fixed the tests. I'll push up again.